packages/localizations/src/ar-SA.ts

```diff
@@ -305,6 +305,10 @@ export const arSA: LocalizationResource = {
       title: 'التحقق من البريد الإلكتروني',
     },
     emailLink: {
+      clientMismatch: {
```

Comment:
❓ Are these changes related to connected accounts?

Reply:
No, i had to run the script that autogenerates all other localization with `undefined` but seems like when `clientMismatch` was added we never run this. It will be a lot manual work to revert them, and they are not causing any harm.